content/advent-2015/semver.md

```diff
@@ -0,0 +1,150 @@
++++
+author = ["Matt Farina"]
+date = "2015-12-02T08:00:00+00:00"
+title = "Working with Semantic Versions"
+series = ["Advent 2015"]
+draft = true
++++
+
+[Semantic Versioning](http://semver.org/) (a.k.a SemVer) has become a popular way
+to handle versions. The structure not only allows for incremental releases but
+allows people and automation to deduce what those changes mean. This makes SemVer
+ideal for a wide range of uses even though they are most well known for package
+managers.
+
+Before we look at how we can work with them in Go let's take a look at what a
+semantic version looks like.
+
+![Semantic Version](/postimages/advent-2015/semver.png)
+
+
+The diagram shows all the parts of the parts of a semantic version. Quite often
+you'll see just the first 3 numbers separate by a `.`. A general breakdown of a
+semantic version is:
+
+- The major number is incremented when the API to the package or application
+  changes in backwards incompatible ways.
+- The minor number is incremented when new features are added to the API without
+  breaking backwards compatibility. If the major number is incremented the
+  minor number returns to 0.
+- The patch number is incremented when no new features are added but bug fixes
+  are released. If the major or minor numbers are incremented this returns to 0.
+- A pre-release is a `.` separated list of identifiers following a `-`. For
+  example, `1.2.3-beta.1`. These are optional and are only needed for pre-release
+  versions. In this case `1.2.3` would be a release version and newer.
```

Comment:
"release version and newer." ... seems to be missing something. 

maybe reiterate what beta.1 is?